org.eclipse.xtext.xtext.ui.examples/projects/arithmetics/org.eclipse.xtext.example.arithmetics/src-gen/org/eclipse/xtext/example/arithmetics/arithmetics/Import.java

```diff
@@ -38,12 +38,12 @@
    * </p>
    * <!-- end-user-doc -->
    * @return the value of the '<em>Module</em>' reference.
-   * @see #setModule(org.eclipse.xtext.example.arithmetics.arithmetics.Module)
+   * @see #setModule(Module)
```

Comment:
Local package has higher precedence than java.lang?

Reply:
no idea what java does here will fix back

Reply:
maybe this is cause of inconsistent emf versions.
time to do that damn emf update